semcore/link/src/style/link.shadow.css

```diff
@@ -88,9 +91,14 @@ SText {
   transition: border-bottom-color 0.15s ease-in-out;
 }
 
-SLink[color] {
-  &:hover {
-    color: var(--colorHoverText);
+SLink[text-color] {
+  color: var(--text-color);
+
+  &[active],
+  &:hover,
+  &:active {
+    color: var(--text-color);
+    filter: brightness(0.8);
```

Comment:
Can we use token for the Link here instead of filter? Token called `--intergalactic-text-link-hover-active`. It's needed to be consistent with the design, and to simplify palette developing for the other themes (for example, dark).